contrib/man2pdf/man2pdf.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
+
+# Copyright (c) 2017 Intel Corp.  All rights reserved.
+#
+# This software is available to you under a choice of one of two
+# licenses.  You may choose to be licensed under the terms of the GNU
+# General Public License (GPL) Version 2, available from the file
+# COPYING in the main directory of this source tree, or the
+# BSD license below:
+#
+#     Redistribution and use in source and binary forms, with or
+#     without modification, are permitted provided that the following
+#     conditions are met:
+#
+#      - Redistributions of source code must retain the above
+#        copyright notice, this list of conditions and the following
+#        disclaimer.
+#
+#      - Redistributions in binary form must reproduce the above
+#        copyright notice, this list of conditions and the following
+#        disclaimer in the documentation and/or other materials
+#        provided with the distribution.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+# EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+# MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+# NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS
+# BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN
+# ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+# CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+#
+# MAN 2 PDF -- Simple script to convert the libfabric manpages into a PDF
+#
+# Requires libfabric manpages to be installed and available in MANPATH.
+# Uses man and ps2pdf to convert each manpage, followed by Mac join.py or
+# pdfjoin to generate final PDF
+#
+
+# Generate list of manpages, but exclude manpages that are symlinks to other manpages
+#MANPAGES=( `echo fabric && man -a -w fi_* | xargs basename | sort | uniq | cut -d . -f 1` )
```

Comment:
I would think that if the goal is to have a searchable set of man pages, the order that you have listed below is not that important.  I think it would be more important to have a script that self-maintains over time (i.e., that we don't introduce yet another list that needs to be maintained over time).